src/hb-blob.cc

```diff
@@ -520,6 +524,36 @@ _hb_mapped_file_destroy (void *file_)
 }
 #endif
 
+#ifdef _PATH_RSRCFORKSPEC
+static int
+_open_resource_fork(const char *file_name, hb_mapped_file_t *file)
+{
+  size_t name_len = strlen (file_name) + sizeof (_PATH_RSRCFORKSPEC);
+  char *rsrc_name = (char *) malloc (name_len);
+  if (unlikely (!rsrc_name)) return -1;
+
+  strncpy (rsrc_name, file_name, strlen (file_name));
+  strncat (rsrc_name, _PATH_RSRCFORKSPEC, sizeof (_PATH_RSRCFORKSPEC) - 1);
```

Comment:
Since you know how many bytes you copied, you can make this faster by not using `cat`.